docs/intro/install.rst

```diff
@@ -34,6 +32,19 @@ To install using pip::
 Platform specific installation notes
 ====================================
 
+Anaconda
+--------
+
+.. note::
+
+  For Windows and OS X users, this is the recommended way to install Scrapy.
```

Comment:
I agree that it's probably better to stick with pip for OS X. Maybe we could recommend this method to anyone who had problems install Scrapy with pip, regardless of the platform (and say it explicitly in this note)?